recipe/meta.yaml

```diff
@@ -20,6 +21,8 @@ requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
     - nose
     - six
```

Comment:
isn't `six` just for runtime?

Reply:
Hmm, that was in the original recipe.  I clearly didn't check.